### PR TITLE
Navigator and Navigator Plus bundles

### PR DESCRIPTION
**Pull request by Airom42**
# Added two new modules

**What's new**
 - Navigator and Navigator plus modules
 - documentation for the modules

@airom42

### CODE REVIEW

File: bundles/navigator/ISAN-navigator_bundle.yolol [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+g=p-:a g*=g gr=(g-gl)/4 gl=g x/=:a :at=z x=10 gotopj//  v2.5n - 190821
+f=p-:a f*=f hr=(f-hl)/4 hl=f x/=:a :at=mr x=7 gotopj
+st=:_!=c :_=rr+"\n\n   "+(:at-"origin_")+sl goto 11*(:a==0)*st
+st=:_!=c :_=rr+"\n\n can't see\n recievers" goto 12*st
+t+=er i+=fr g+=gr f+=hr //ISAN doc: isan.to/doc   -   Starmap: isan.to
+x+=(6-x)*(x<7) xx=(t+i)/e+g/u-f/k yy=i/l-t/mm+(g+f)/nn zz=(g+f-t-i)/o
+st=:_==c :_=rr+h+xm+xx/n*A+ym+yy/n*A+zm+zz/n*A+sm+s gotoej+x*so+p*st
+w+=xx uu+=yy vv+=zz r=w-tu j=uu-uv v=vv-vt ej+=(ii++%3)>1 gotox
+m=(r*r+j*j+v*v)^b s=m/div*A tu=w uv=uu vt=vv w=0 uu=0 vv=0 ej=16 y=:n
+d-=xx y-=yy q=:o-zz :f=(d*d+y*y+q*q)^b :e=(r*d+j*y+v*q)*c/:f/m+c gotox
```
[the remark this answers is not in the document]
Comment (Evanito) [reply bundles/navigator/ISAN-navigator_bundle.yolol:19]: Was able to correctly calculate f using:
`d=:m-xx y=:n-yy q=:o-zz :f=(d*d+y*y+q*q)^b`
Yet still had issues with `e`

**Review by Evanito — COMMENTED** [review]
Cool project, much needed!

Couldn't get working, even when dove into the code. 
Could use explanation of the "Alignment" feature, and what the `e` display shows (when/if it works)

File: bundles/navigator/ISAN-navigator_bundle.yolol (hunk 1 above)
[the remark this answers is not in the document]
Comment (Evanito) [reply bundles/navigator/ISAN-navigator_bundle.yolol:19]: Note that the exception occurs even when `:f` is nonzero

**Comment by Azurethi** [comment]
> Cool project, much needed!
> 
> Couldn't get working, even when dove into the code.
> Could use explanation of the "Alignment" feature, and what the `e` display shows (when/if it works)

Closed this as it was not implemented correctly. You might like to check out this: https://github.com/Collective-SB/ISAN/tree/master/bundles/navigator

**Pull request closed without merging**